### ISSUE TITLE
Translatable menu items

### ISSUE THREAD

**Issue by prescott66** [manjaro/manjaro-settings-manager#73]
It would be nice to have ability to translate menu items, as another menu items in settings menu in XFCE. I mean items "Manjaro Notifier Settings" and "Manjaro Settings Manager".

**Comment by scachemaille** [comment]
the menu translation are done in the differents .desktop files 
ex:
```
[Desktop Entry]
Name=Manjaro Settings Manager
Name[de]=Manjaro Einstellungen
Name[en_GB]=Manjaro Settings Manager
Name[fr]=Manjaro Gestionnaire de paramètres
Name[ro]=Gestionar de Setări Manjaro

Comment=Graphical Settings Manager for Manjaro
Comment[de]=Grafische Einstellungsverwaltung für Manjaro
Comment[en_GB]=Graphical Settings Manager for Manjaro
Comment[fr]=Gestionnaire graphique de paramètres pour Manjaro
Comment[ro]=Gestionar grafic de setări de sistem pt. Manjaro

Exec=manjaro-settings-manager -style=gtk
Icon=preferences-desktop
Terminal=false
Type=Application
Categories=Settings;DesktopSettings;X-XFCE-SettingsDialog;X-XFCE-SystemSettings;
OnlyShowIn=XFCE;GNOME;Unity;X-Cinnamon;MATE;
```

**Comment by prescott66** [comment]
thank you for help! i have created pull request https://github.com/manjaro/manjaro-settings-manager/pull/75  for this issue.after merging, this issue can be closed :wink:

**Comment by scachemaille** [comment]
fixed with #75
